@memZono/memZono.m

```diff
@@ -80,308 +91,278 @@
+        % Transformations ---------------------------
+        obj = map(obj1,obj2,inDims,outDims); % Maping function
+        obj = transform(obj1,obj2,M,inDims,outDims); % Mapping w/ dims (outdated/to be replaced)
+        function out = affine(in,M,b,inDims,outDims) %% Affine (technically not needed but useful version instead of map version directly)
+            out = in(inDims).transform(b,M,inDims,outDims);
         end
+        % Plotting -------------------------
+        plot(obj, dims, varargin);
+        % Projection & Export ------------------
+        out = projection(obj,dims); % projection according to dims
+        function out = Z(obj,dims), out = projection(obj,dims).Z_; end
+        % Additional methods (implimentations of abstractZono methods in memZono)
+        [NN,Y] = reluNN(X,Ws,bs,a);
+        [s,x_out] = supportFunc(obj,dims,d_in)
     end
 
-    %% In/Out with base Zonotope
+    %% Indexing  ----------------------------
     methods
-        % test if special
-        function out = issym(obj)
-            % tests if any are symbolic
-            if any([isa(obj.G,'sym'),isa(obj.c,'sym'),...
-                    isa(obj.A,'sym'),isa(obj.b,'sym')])
-                out = true;
-            else
-                out = false;
+        B = subsref(A,S);
+        % A = subsasgn(A,S,B); %<---- not overloaded
+    end
+
+    %% Overrides ----------------------------
+    methods
+        % +, plus() - override for memorySum()
+        function out = plus(in1,in2), out = in1.memorySum(in2); end
+        % *, mtimes() - override for transform (map?) ... mainly just scalers
+        function out = mtimes(in1,in2)
+            if isa(in2,'memZono'), out = in2.transform([],in1); %<== flip the syntax order
+            else, error("mtimes only overloaded for left multiplication");
             end
         end
-        function out = isnumeric(obj)
-            % tests if all are numeric
-            if all([isnumeric(obj.G), isnumeric(obj.c), ...
-                    isnumeric(obj.A), isnumeric(obj.b)])
-                out = true;
-            else
-                out = false;
-            end
+        % &, and() - overide memoryIntersection w/ checks
+        function out = and(obj1,obj2,sharedDimLabels)
+            if nargin == 2, error('Must Supply labels for shared dimensions'); end
+            out = memoryIntersection(obj1,obj2,sharedDimLabels);
         end
-
-        function out = get.baseClass(obj)
-            if all(obj.vset)
-                if isempty(obj.A_)
-                    out = 'zono';
-                else
-                    out = 'conZono';
-                end
-            else
-                out = 'hybZono';
-            end
+        % Overide for memoryUnion w/ checks ... NOT IMPLIMENTED
+        function obj = or(obj1,obj2), error('Union not yet implimented'); end
+        % vertcat (Extended cartProd)
+        function obj = vertcat(varargin)
```

Comment:
From @jruths:
We can eliminate the warning... this would require identifying all the shared dims/factors/cons between each combination and doing them in one operation... a bunch of combinatorics and not explicitly defined

Reply:
We can eliminate the warning... this would require identifying all the shared dims/factors/cons between each combination and doing them in one operation... a bunch of combinatorics and not explicitly defined